catalog-info.yaml

```diff
@@ -21,3 +21,32 @@ spec:
     spec:
       repository: elastic/terraform-provider-ec
       pipeline_file: ".buildkite/pipeline.yml"
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-terraform-provider-ec-release
+  description: Buildkite Pipeline for releasing the terraform-provider-ec
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/terraform-provider-ec-release
+
+spec:
+  type: buildkite-pipeline
+  owner: group:control-plane-stateful
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: terraform-provider-ec-release
+    spec:
+      repository: elastic/terraform-provider-ec
+      pipeline_file: ".buildkite/release.yml"
+      provider_settings:
+        build_branches: false
+        build_pull_request_forks: false
+        build_tags: true
+        filter_condition: 'build.tag =~ /^v[0-9.]+$/'
+        filter_enabled: true
```

Comment:
Do we need to add [`teams`](https://github.com/elastic/terraform-provider-elasticstack/blob/main/catalog-info.yaml#L25) as well? I added it to Elasticstack provider, also I wanted to grant @tobio the perms to release. Toby is in the `serverless-core-architecture`. I think it makes sense to grant Toby such permissions here too.

Reply:
Yes we do - I mirrored the teams with GitHub access (see [RRE](https://github.com/elastic/catalog-info/pull/635/files#diff-9f7cd1a723120f0178ff1ce9b1f671ca39c0bb61fc60492541f52630c9b5c0acR28-R39))